oaknational/Oak-Web-Application
# Added "why this why now" and "description" to unit modal

## Description
Added unit "description" and "why this why now" for cycle 2 subjects into the UI 

## Issue(s)

Fixes `CUR-832`/`CUR-831`

## How to test
For cycle 1:

 1. Head to curriculum visualiser
 2. Check prior/future unit description is same as previous
 
Enable cycle 2 units

 1. Check "description" and "why this why now" is present.

## Screenshots
Cycle 1
<img width="1169" alt="Screenshot 2024-09-19 at 15 29 07" src="https://github.com/user-attachments/assets/228798a4-3a5e-4d50-a871-1c0375dc5461">

Cycle 2
<img width="1169" alt="Screenshot 2024-09-19 at 15 28 48" src="https://github.com/user-attachments/assets/4d3c2d83-fe11-404f-bf97-0c4c9d6ca36e">
